Sources/AppStoreConnectCLI/Model/BetaTester.swift

```diff
@@ -5,74 +5,90 @@ import Combine
 import Foundation
 import SwiftyTextTable
 
-extension BetaTester: ResultRenderable { }
+struct BetaTester: ResultRenderable {
+    let email: String?
```

Comment:
It would be nice if we could define what was optional and what was non optional but we can do a pass over that in a different PR